ci/build-pbs-packages.sh

```diff
@@ -0,0 +1,42 @@
+#! /bin/bash -xe
+
+. /etc/os-release
+
+pbsdir=/pbssrc
+rpm_dir=/root/rpmbuild
+
+mkdir -p /src/packages
+mkdir -p ${rpm_dir}/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
+
+cp -r $pbsdir /tmp/pbs
+cd /tmp/pbs
+./autogen.sh
+./configure --prefix=/opt/pbs --enable-ptl
```

Comment:
shall we also provide CFLAGS?

Reply:
We were not doing this in the docker folder, nor do we specify so in INSTALL file of the repo. Is there a reason we should add CFLAGS here and if so which ones?

Reply:
in our travis builds I see "-g -O2 -Wall -Werror"